fiware-region-sanity-tests/README.rst

```diff
@@ -241,21 +241,33 @@ rest of standard parsers bundled in NGSI Adapter package.
 
 
 D-Bus and HTTP PhoneHome Service for E2E tests
----------------------------------------------
+----------------------------------------------
 
 Some E2E test cases have been implemented to check the connection in both
 *Internet -> VM* and *VM -> Internet*.
-This test cases are:
+These test cases are:
 
 * Test whether it is possible to deploy an instance, assign an allocated
   public IP and establish a SSH connection *(Internet -> VM)*
 * Test whether it is possible to deploy an instance
   and connect to INTERNET (SNAT) without assigning a public IP *(VM -> Internet)*
 
-The later will try to execute a *PhoneHome request* (executed by Cloud-Init in the VM)
+The latter will try to execute a *PhoneHome request* (executed by Cloud-Init in the VM)
 to the *HTTP PhoneHome service* running in the configured HOST:PORT
 (*phonehome_endpoint* configuration). If this value is not set, this test will be skipped.
 
+There is another test implemented:
+
+* Test whether it is possible to deploy an instance and check that metadata service is working properly.
+
+This test is checking if openstack metadata service is working. It gets the metadata information about
+the VM deployed and return to the HTTP phonehome server.
+
```

Comment:
what about: "and sends metadata back to the PhoneHome server to check whether is it correct"?